src/Lorentz/Contracts/GameDAO.hs

```diff
@@ -18,12 +18,23 @@ module Lorentz.Contracts.GameDAO
   , config
   ) where
 
-import Universum hiding ((>>), drop, swap)
 import Lorentz
+
 import qualified Lorentz.Contracts.BaseDAO as DAO
 import qualified Lorentz.Contracts.BaseDAO.Types as DAO
+import Universum hiding (drop, swap, (>>))
 import Util.Markdown
 
+data GameDaoContractExtra = GameDaoContractExtra
```

Comment:
Is it better than `()`?

Reply:
I wanted to add something interesting there but forgot. Gonna add now.